<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item row="2" column="0">
         <widget class="FileTree" name="treeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_name_sheet">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>18</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit_name_script">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>18</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QTabWidget" name="tabWidget">
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab_editor">
         <attribute name="title">
          <string>&amp;Editor</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="TextEdit" name="textEdit_editor">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_table">
         <attribute name="title">
          <string>&amp;Table</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QWidget" name="widget_5" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>10</y>
               <width>31</width>
               <height>21</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ROW</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_sheet_rowp">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>10</y>
               <width>41</width>
               <height>21</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_sheet_rowm">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>10</y>
               <width>41</width>
               <height>21</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>270</x>
               <y>10</y>
               <width>31</width>
               <height>21</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>COL</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_sheet_colp">
             <property name="geometry">
              <rect>
               <x>220</x>
               <y>10</y>
               <width>41</width>
               <height>21</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_sheet_colm">
             <property name="geometry">
              <rect>
               <x>310</x>
               <y>10</y>
               <width>41</width>
               <height>21</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_sheet_clear">
             <property name="geometry">
              <rect>
               <x>410</x>
               <y>10</y>
               <width>75</width>
               <height>21</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>clear</string>
             </property>
            </widget>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="TableWidget" name="tableWidget_table"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_output">
         <attribute name="title">
          <string>&amp;Output</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="TextBrowser" name="textBrowser_output">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_setting">
         <attribute name="title">
          <string>&amp;Setting</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QToolBox" name="toolBox">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>618</width>
               <height>413</height>
              </rect>
             </property>
             <attribute name="label">
              <string>TextEditor</string>
             </attribute>
             <widget class="QLineEdit" name="lineEdit_setE_test">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>101</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>test   123</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QLabel" name="label_set4">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>40</y>
                <width>91</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>・文字サイズ</string>
              </property>
             </widget>
             <widget class="QSpinBox" name="spinBox_setE_fs">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>40</y>
                <width>42</width>
                <height>20</height>
               </rect>
              </property>
              <property name="value">
               <number>9</number>
              </property>
             </widget>
             <widget class="QLabel" name="label_set5">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>60</y>
                <width>91</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>・フォント</string>
              </property>
             </widget>
             <widget class="QFontComboBox" name="fontComboBox_setE_f">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>60</y>
                <width>231</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_set6">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>80</y>
                <width>91</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>・タブ幅</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="lineEdit_setE_ts">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>80</y>
                <width>113</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>40.0000</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_set7">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>100</y>
                <width>91</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>・文字の折り返し</string>
              </property>
             </widget>
             <widget class="QCheckBox" name="checkBox_setE_wp">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>100</y>
                <width>75</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QLabel" name="label_set8">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>120</y>
                <width>91</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>・斜体</string>
              </property>
             </widget>
             <widget class="QCheckBox" name="checkBox_setE_ir">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>120</y>
                <width>75</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QLabel" name="label_set9">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>140</y>
                <width>91</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>・ボールド</string>
              </property>
             </widget>
             <widget class="QCheckBox" name="checkBox_setE_bl">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>140</y>
                <width>75</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="page_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>618</width>
               <height>413</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Gnuplot</string>
             </attribute>
             <widget class="QLabel" name="label_set11">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>111</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>・実行ファイルのパス</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="lineEdit_setG_p">
              <property name="geometry">
               <rect>
                <x>180</x>
                <y>10</y>
                <width>321</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>gnuplot.exe</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_set11_2">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>40</y>
                <width>111</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>・事前実行コマンド</string>
              </property>
             </widget>
             <widget class="TextEdit" name="textEdit_setG_precmd">
              <property name="geometry">
               <rect>
                <x>180</x>
                <y>40</y>
                <width>321</width>
                <height>101</height>
               </rect>
              </property>
              <property name="lineWrapMode">
               <enum>QPlainTextEdit::NoWrap</enum>
              </property>
             </widget>
             <widget class="QLabel" name="label_set11_3">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>150</y>
                <width>151</width>
                <height>22</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>・sheetプロット時のオプション</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="lineEdit_setG_sheetplotop">
              <property name="geometry">
               <rect>
                <x>180</x>
                <y>150</y>
                <width>321</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>with linespoints pointtype 2 pointsize 2</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_help">
         <attribute name="title">
          <string>&amp;Help</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_excute">
        <property name="text">
         <string>実行</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+R</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="CmdLine" name="lineEdit_cmdline"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>tablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>TextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>textbrowser.h</header>
  </customwidget>
  <customwidget>
   <class>CmdLine</class>
   <extends>QLineEdit</extends>
   <header>cmdline.h</header>
  </customwidget>
  <customwidget>
   <class>FileTree</class>
   <extends>QTreeWidget</extends>
   <header>filetree.h</header>
  </customwidget>
  <customwidget>
   <class>TextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>textedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
